packages/shared/src/features/common/components/FormInputCheckboxGroup.tsx

```diff
@@ -72,8 +72,9 @@ const FormInputCheckbox = ({
       aria-checked={isSelected}
       aria-label={item.label}
       className={classNames(isVertical ? 'typo-subhead' : 'typo-body')}
-      pressed={isSelected}
+      data-track-click
```

Comment:
What is this actually? not familiar with it.

Reply:
I've added a onClickCapture on stepper level, that send events on each element that has this `data-track-click`